test/fork/airstream/CampaignLL.t.sol

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.19 <0.9.0;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { Lockup, LockupLinear } from "@sablier/v2-core/src/types/DataTypes.sol";
+
+import { ISablierV2AirstreamCampaignLL } from "src/interfaces/ISablierV2AirstreamCampaignLL.sol";
+
+import { MerkleBuilder } from "../../utils/MerkleBuilder.sol";
+import { Fork_Test } from "../Fork.t.sol";
+
+abstract contract AirstreamCampaignLL_Fork_Test is Fork_Test {
+    constructor(IERC20 asset_) Fork_Test(asset_) { }
+
+    function setUp() public virtual override {
+        Fork_Test.setUp();
+    }
+
+    /// @dev Encapsulates the data needed to compute a Merkle tree leaf.
+    struct LeafData {
+        uint256 index;
+        address recipient;
+        uint128 amount;
+    }
+
+    struct Params {
+        address admin;
+        uint40 expiration;
+        LeafData[] leafData;
+        uint256 leafPos;
+    }
+
+    struct Vars {
+        uint256 actualAirstreamId;
+        LockupLinear.Stream actualStream;
+        uint128[] amounts;
+        ISablierV2AirstreamCampaignLL campaignLL;
+        uint256 campaignTotalAmount;
+        uint128 clawbackAmount;
+        uint256 recipientsCount;
+        uint256 expectedAirstreamId;
+        address expectedCampaignLL;
+        LockupLinear.Stream expectedStream;
+        uint256[] indexes;
+        bytes32[] leaves;
+        bytes32 merkleRoot;
+        address[] recipients;
+    }
+
+    function testForkFuzz_AirstreamCampaignLL(Params memory params) external {
+        vm.assume(params.admin != users.admin.addr);
```

Comment:
We need two additional checks:

- No zero address
- No blacklisted

Note: we don't need the blacklisted check for the recipient because we're using the default account (Alice?) when claiming.

Reply:
>Note: we don't need the blacklisted check for the recipient because we're using the default account (Alice?) when claiming.

Not really, we don't need because when the clam is made, the erc20 tokens are going on the either `lockupLinear` or `lockupDynamic` contract, we are not testing `withdraw` function, this is why it doesn't matter.

Reply:
The claim is made from the address found at the fuzzed param `leafPos`

Reply:
You are right.